react-app/src/library/services/vendor-implementations/jabra/jabra.ts

```diff
@@ -434,8 +434,24 @@ export default class JabraService extends VendorImplementation {
   }
 
   async disconnect (): Promise<void> {
-    this.headsetEventSubscription && this.headsetEventSubscription.unsubscribe();
-    (this.isConnected || this.isConnecting) &&
-      this.changeConnectionStatus({ isConnected: false, isConnecting: false });
+    try {
+      if (!this.callLock) {
+        return this.logger.info(
+          'Currently not in possession of the Call Lock; Cannot react to Device Actions'
+        );
+      }
+      this.callControl.releaseCallLock();
+    } catch ({ message, type }) {
+      if (this.checkForCallLockError(message, type)) {
+        this.logger.info(message);
```

Comment:
I'm not familiar enough with headsets, but why is this `this.logger.info` instead of `this.logger.error`?

Reply:
So when you try to `takeCallLock` but you already are in possession or you try to `releaseCallLock` but you aren't in possession, the Jabra SDK will throw an error explaining the respective scenario.  However, in our case, we don't necessarily see those cases as errors in context.  It's more so like "hey we're trying to take the call lock. oh we already have it? okay! we can then proceed with our steps since we already have.  we should also just log that we were in possession already."  It also allows us to differentiate between that kind of error or an actual error where something went wrong during the process.  Hopefully I explained that well but let me know if not!